pallets/msa/src/offchain_storage.rs

```diff
@@ -0,0 +1,464 @@
+use crate::{pallet::OffchainIndexEventCount, Config, Event, Pallet, PublicKeyToMsaId};
+pub use common_primitives::msa::MessageSourceId;
+/// Offchain Storage for MSA
+use common_primitives::offchain::{
+	self as offchain_common, get_msa_account_lock_name, get_msa_account_storage_key_name,
+	LockStatus, MSA_ACCOUNT_LOCK_TIMEOUT_EXPIRATION,
+};
+use frame_support::RuntimeDebugNoBound;
+use frame_system::pallet_prelude::BlockNumberFor;
+use parity_scale_codec::{Decode, Encode};
+use sp_core::serde::{Deserialize, Serialize};
+extern crate alloc;
+use alloc::string::String;
+use sp_io::offchain_index;
+use sp_runtime::{
+	offchain::{
+		storage::StorageValueRef,
+		storage_lock::{BlockAndTime, StorageLock, Time},
+		Duration,
+	},
+	traits::One,
+	Saturating,
+};
+use sp_std::{collections::btree_map::BTreeMap, fmt::Debug, vec, vec::Vec};
+
+/// Block event storage prefix
+const BLOCK_EVENT_KEY: &[u8] = b"frequency::block_event::msa::";
+
+/// Offchain index for MSA events count
+const BLOCK_EVENT_COUNT_KEY: &[u8] = b"frequency::block_event::msa::count::";
+
+/// Lock expiration timeout in in milli-seconds for initial data import msa pallet
+const MSA_INITIAL_LOCK_TIMEOUT_EXPIRATION: u64 = 3000;
+
+/// Lock expiration block for initial data import msa pallet
+const MSA_INITIAL_LOCK_BLOCK_EXPIRATION: u32 = 30;
```

Comment:
what are the units for this--sec or msec? Add units to const name.

Reply:
renamed